src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -84,7 +84,7 @@ partial class ApplicationEngine
             [OpCode.REVERSE4] = 60,
             [OpCode.REVERSEN] = 400,
             [OpCode.INITSSLOT] = 400,
-            [OpCode.INITSLOT] = 800,
+            [OpCode.INITSLOT] = 1600,
```

Comment:
Why is the price of `INITSLOT` 4 times that of `INITSLOT`?

Reply:
Compared with `INITSSLOT`, the logic of INITSLOT contains another loop, which can at most be run 255 times:
https://github.com/neo-project/neo-vm/blob/a14b934f5b5c8779164e3b96b276984b3a57ccd4/src/neo-vm/ExecutionEngine.cs#L491